docs/pages/workflow/customizing.mdx

```diff
@@ -3,73 +3,92 @@ title: Add custom native code
+  description="A tutorial on creating a native module that persists settings with the Expo Modules API."
+  href="/modules/native-module-tutorial/"
+  Icon={BookOpen02Icon}
+/>
 
-### Revert changes from `npx expo run:[android|ios]`
+<BoxLink
+  title="Tutorial: Creating a native view"
+  description="A tutorial on creating a native view that renders a native WebView component with the Expo Modules API."
+  href="/modules/native-view-tutorial/"
+  Icon={BookOpen02Icon}
+/>
 
-Suppose you have decided to roll your app back to being fully managed (no **android** or **ios** directories in your project directory). In that case, you can check out your most recent commit before executing `npx expo run:[android|ios]`, then run `npm install` again to restore the state of your **node_modules** directory.
+### Creating a local module
 
-### Develop apps with custom native code
+If you intend to use your native module in a single app (you can always change your mind later), we recommend [using a "local" Expo module](/modules/get-started/#creating-the-local-expo-module) to write custom native code. Local Expo Modules function similarly to [Expo Modules](/modules/overview) used by library developers and within the Expo SDK, like `expo-camera`, but they are not published on npm. Instead, you create them directly inside your project.
 
-Once you have customized the native code in your project, you can use the [`expo-dev-client`](/develop/development-builds/introduction/#what-is-an-expo-dev-client) library to [create a development build](/develop/development-builds/create-a-build/) and retain the convenience of working with JavaScript or TypeScript in Expo Go.
+Creating a local module scaffolds a Swift and Kotlin module inside the `modules` directory in your project, and these modules are automatically linked to your app.
 
-## Release apps with custom native code to production
+<Terminal cmd={['npx create-expo-module@latest --local', 'npx expo run']} />
 
-When you're ready to ship your app, you can [build it with EAS Build](/build/introduction) the same as you were building it before adding custom native code. Alternatively, you can archive and sign it locally. Unsurprisingly, we recommend EAS Build!
+### Sharing a module with multiple apps
 
-<Terminal
-  cmd={['# Install the CLI', '$ npm i -g eas-cli', '', '# Build your app!', '$ eas build -p all']}
-  cmdCopy="npm i -g eas-cli && eas build -p all"
-/>
+If you intend to use your native module with multiple apps, then use `npx create-expo-module@latest,` leave out the `--local` flag, and [create a standalone module](/modules/use-standalone-expo-module-in-your-project/). You can publish your package to npm, or you can put it in a packages directory in your [monorepo](/guides/monorepos/) (if you have one) to use it in [a similar way to local modules](/modules/use-standalone-expo-module-in-your-project/).
+
+### Considerations when using Continuous Native Generation (CNG)
```

Comment:
```suggestion
## Considerations when using Continuous Native Generation (CNG)
```

This is an H2 since the following sections are subsections of this section and it is independent of the previous two sections (as per my observation on reading this page).